modules/plottinggl/src/plotters/scatterplotgl.cpp

```diff
@@ -409,19 +410,9 @@ void ScatterPlotGL::plot(const size2_t& dims, IndexBuffer* indexBuffer, bool use
         if (selectedIndicesGLDirty_) {
             nSelectedButNotFiltered_ = 0;
             std::vector<uint32_t> selectedIndices;
-            if (indexColumn_) {
-                auto& indexCol =
-                    indexColumn_->getTypedBuffer()->getRAMRepresentation()->getDataContainer();
-                for (auto [ind, selected, filtered] : util::enumerate(selected_, filtered_)) {
-                    if (selected && !filtered) {
-                        selectedIndices.push_back(indexCol[ind]);
-                    }
-                }
-            } else {
-                for (auto [ind, selected, filtered] : util::enumerate(selected_, filtered_)) {
-                    if (selected && !filtered) {
-                        selectedIndices.push_back(static_cast<uint32_t>(ind));
-                    }
+            for (auto [ind, selected, filtered] : util::enumerate(selected_, filtered_)) {
```

Comment:
brushing and linking contain the identifier (not the row)?

Reply:
ScatterPlotGL only uses zero-based indices, i.e. row numbers. The translation to IDs is handled by the ScatterPlot processor.

Reply:
They always have. Compare for example with the PCP
https://github.com/inviwo/inviwo/blob/7fdcf8a8e65dc9c996f15e5c0649545171fcb7af/modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp#L490-L502